src/types.ts

```diff
@@ -33,18 +33,32 @@ type IframeConnectedMessage = {
   action: 'IFRAME_CONNECTED';
   connected: true;
   tags: number;
+  url: string;
```

Comment:
I don't feel super strongly but as a suggestion, maybe this could be `root` or `anchorUrl` for clarity?